Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -240,6 +242,8 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     auto lastSeenBaseURL = URL { objectForKey<NSString>(m_state, lastSeenBaseURLStateKey) };
     [m_state setObject:(NSString *)m_baseURL.string() forKey:lastSeenBaseURLStateKey];
 
+    [m_state setObject:extension().displayName() forKey:lastSeenDisplayNameStateKey];
```

Comment:
this is what's causing the test failures on the PR. 
`if (NSString *displayName = extension().displayName())
        [m_state setObject:displayName forKey:lastSeenDisplayNameStateKey];`